.github/workflows/cmake.yml

```diff
@@ -140,6 +140,12 @@ jobs:
     with:
       name: L0
 
+  level-zero-v2:
+    name: Level Zero V2
+    uses: ./.github/workflows/build-hw-reusable.yml
+    with:
+      name: L0_V2
```

Comment:
this will not run, there are no such runners.

Reply:
Right, done.